<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>c4psy program</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/psy.png</normaloff>icons/psy.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWebView" name="main_text">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>Вопрос %v из %m</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="h_lay">
      <item>
       <widget class="QPushButton" name="bt_prev">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Предыдущий вопрос</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/back.png</normaloff>icons/back.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_multi">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_next">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Следующий вопрос</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/foward.png</normaloff>icons/foward.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="button_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Тест</string>
    </property>
    <addaction name="act_new_test"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="act_about"/>
    <addaction name="act_about_qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="act_new_test">
   <property name="icon">
    <iconset>
     <normaloff>icons/filenew.png</normaloff>icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>Начать тест</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="icon">
    <iconset>
     <normaloff>icons/cancel.png</normaloff>icons/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="act_about">
   <property name="icon">
    <iconset>
     <normaloff>icons/help.png</normaloff>icons/help.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="act_about_qt">
   <property name="icon">
    <iconset>
     <normaloff>icons/qt.png</normaloff>icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>PyQt4.QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>act_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
